restapi/api.go

```diff
@@ -21,10 +25,31 @@ func Router(db *gorm.DB) (http.Handler, error) {
 	}
 
 	rest.GetHandler(r, "/health", api.health)
-
+	rest.PostHandler(r, "/authors", api.authorCreate)
 	return r, nil
 }
 
+func (api api) authorCreate(w http.ResponseWriter, r *http.Request) render.Renderer {
```

Comment:
If you notice code that is merged in master, I told megan to move her code into `stories.go` file. Can you do the same and move endpoints code into `authors.go`. This way `api.go` won't` be huge